src/als/ui/dialogs.py

```diff
@@ -60,6 +65,11 @@ def _show_missing_folders(self):
             else:
                 ui_field.setStyleSheet("border: 1px")
 
+        if not Path(self._ui.ln_master_dark_path.text()).is_file():
+            self._ui.ln_master_dark_path.setStyleSheet("border: 1px solid orange")
+        else:
+            self._ui.ln_master_dark_path.setStyleSheet("border: 1px")
+
```

Comment:
Please don't duplicate stylesheet strings.
If one day we decide -- out of nowhere -- to change the warning color, we'll have to do it twice (or even more).

Reply:
And remember : an empy value must be allowed for the dark path

Reply:
Ok, now it tests if chk_use_dark is checked and then it verifies if the path is correct. Dark are used only if chk_use_dark is checked so the verification is need there.

Reply:
Imagine the following scenario 

- Last time ALS was used, dark file path was correct, but user chosed not to do dark subtraction and unchecked the box
- User exited ALS and did some other things, you know, for life :)
- Configured dark file got deleted from the user computer. These things do happen, you know
- User launches ALS & opens preferences
- As the checkbox is unchecked, the dark file path is not validated and no warning is shown
- User now decides to do dark subtraction and checks the box. She is now confident that no warning being shown, dark subtraction will occur correctly on next session
- User scratches her head as she keeps seeing those pesky colorful trails and eventually realizes something is fishy with the dark file when she hopefully-soon-enough opens preferences again.

I do think we should check the validity of the dark file path whenever said path is non-empty, regardless of the checkbox state.

Or do you have a better idea ?

Reply:
No, you're right :) . 
So I modify the code. Now it verifies path validity systematically if a path is set. Empty path is allowed only if chk_use_dark is not checked.